forecasting/src/autogluon/forecasting/trainer/model_presets/presets.py

```diff
@@ -0,0 +1,147 @@
+import copy
+import inspect
+import logging
+
+import autogluon.core as ag
+
+from ...models.abstract.abstract_model import AbstractModel
+from ...models.gluonts_model.mqcnn import MQCNNModel
+from ...models.gluonts_model.sff import SimpleFeedForwardModel
+from ...models.gluonts_model.deepar import DeepARModel
+from ...models.gluonts_model.auto_tabular import AutoTabularModel  # TODO: this module is not fully prepared.
+
+logger = logging.getLogger(__name__)
+
+
+MODEL_TYPES = dict(
+    MQCNN=MQCNNModel,
```

Comment:
Is there a way to not hard code these so that we can more easily extend to more models?

Reply:
I understand that the intention is that these are "defaults," and will be hardcoded. Otherwise the Predictor always allows keying hyperparameter dictionaries with the types (e.g., `DeepARModel`) that inherit from `AbstractModel` so the models that can be used are not limited to these. For easier extensibility, we can think about a generic adapter around GluonTS `Estimator`s as a next step (currently all models are wrapped one by one).